change/@fluentui-eslint-plugin-5733e325-0c8b-4b50-ad12-d52a70e3d28b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Classify .stories.tsx as examples not tests",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "elcraig@microsoft.com",
+  "dependentChangeType": "none"
```

Comment:
this won't bump all versions of all packages right ?

Reply:
Yup that's to avoid a bunch of unnecessary bumps